core/model/formats/bibtexmisc.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+require('../../../utils/array');
+
+/**
+ * Generates the string that represents the author's name for the bibtex misc entry
+ * @param {Author} author - The Author to generate the name string from
+ * @return An APA representation of the authors name.  Will be null if a name cannot be generated.
+ */
+function getAuthorName(author) {
+	// If the author has a name
+	if(author.lastName != null || author.firstName != null) {
+		// If we have the authors first and last name
+		if(author.lastName && author.firstName) {
+			 return author.lastName + ', ' + author.firstName;
+		}
+		// If we just have the authors first or last name
+		else {
+			return (author.lastName != null) ? author.lastName : author.firstName;
+		}
+	}
+
+	return null;
+}
+
+/**
+ * @module bibTex Misc Entry Formatter
+ * Module for formatting source data into a bibTeX misc type entry
+ */
+
+module.exports = {
+    /**
+	 * Formats a source data object to a citation
+	 * @param {SourceData} sourceData - The SourceData object to generate the citation string from
+	 * @return {string} - The bibTeX Misc Entry
+	 */
+    format : (sourceData) => {
+		let returnString = '@misc{';
+	            
+        // Title Block
+        if(sourceData.name != null) {
+            returnString += sourceData.name.replace(/\s+/g, '');
```

Comment:
This will modify the sourceData.name property.  Do we really want to modify an parameter? Can we make a copy instead?